Blob code tweaks

ChangelingRain opened the pull request:
Me, not knowing about or wanting to bother with merge conflict fixes the proper way!
Old pr #11555 

> Blob shields now actually block atmos thank me later
> Blob space expansion now always applies blob_act() even if it doesn't expand into the space tile, also it makes a sound if it fails so you know it's comin after ya

Makes blob code use a maxhealth var(health system 2 return of the health system)
Makes a var much less useful via health changes(this isn't a buff or a nerf it'll have the same results)
Blob health regen is now also a var.
Adds self to admins.txt

ChangelingRain committed:
Merge pull request #6 from tgstation/master

ah yes, not knowing how to fix merge conflicts properly

ChangelingRain committed:
Blobs can now expand into space, woo woo. Cleans up blob code a touch.

ChangelingRain committed:
And the changelog.

ChangelingRain committed:
I missed like one thing and it was shields blocking atmos

Alek2ander commented:
The 1/4 power space tiles should only count as 1/4 of a tile for the blob victory counter.

Anonus commented:
AH YES, MAGICAL CONFLICTS FROM ABSOLUTELYFUCKINGNOTHING

ChangelingRain committed:
It... says the conflicts are fixed?
But no, I'm in hell!

Anonus commented:
FIXING CONFLICTS, WITH AN EMPTY COMMIT

ChangelingRain committed:
WATCH ME FUCK UP CONFLICT FIXES HORRIBLY
GIT CREATES REVERTS FROM NOTHING, EVERYTHING IS HELL

Anonus commented:
LIFE IS HELL, CONFIRMED

PKPenguin321 commented:
without oxygen blob *can* evoluate

ChangelingRain committed:
FOUR WRENCH HITS >>>> THREE WRENCH HITS
Polled, 3 was the highest option, followed by 1 and 2 being tied.

Anonus commented:
This is ready for merge!

ChangelingRain committed:
Three hits to two hits, now caps maxhealth so that space blobs remain fragile.

ChangelingRain committed:
Let your memes, be dream-mes.
Blobs can no longer easily expand into space so that I can get the code stuff merged, smart plan, me

ChangelingRain committed:
I dream, of memes.

ChangelingRain committed:
Conflict fix meme.

ChangelingRain committed:
Ah yes, conflict fixing is so much fun sorry for all these commits

Anonus commented:
HELLO YES THIS IS STILL MERGE-READY PLEASE MERGE IT BEFORE IT CONFLICTS HARDER

phil235 commented on code/game/gamemodes/blob/theblob.dm:
```diff
@@ -121,12 +121,15 @@
 	if(T.Enter(B,src))//Attempt to move into the tile
 		B.density = initial(B.density)
 		B.loc = T
+		B.update_icon()
```
Why do that change? What's bad about just doing `if(istype(T, /turf/space) && prob(65)) 	return` ?

phil235 commented on code/game/gamemodes/blob/theblob.dm:
```diff
@@ -221,14 +224,23 @@
 	health = 21
+	maxhealth = 25
+	health_regen = 1
+	brute_resist = 4
```
that's quite a buff.

phil235 commented:
Instead of merging directly, I'm closing this and making a copy of this PR that I'll pull so there's less commits merged.
